transform/transformers/PDFTransformer.py

```diff
@@ -85,7 +85,7 @@ def get_elements(self):
         return elements
 
     def get_localised_date(self, date_to_transform, timezone='Europe/London'):
-        return arrow.get(date_to_transform).to(timezone).format("ddd DD MMM HH:mm:ss YYYY")
+        return arrow.get(date_to_transform).to(timezone).format("DD MMMM YYYY HH:mm:ss")
```

Comment:
Do we want to be changing the format to be different to what is currently in sdx-collect? I kept it as the original.